tools/azure-pipelines/jobs-template.yml

```diff
@@ -177,6 +145,8 @@ jobs:
     - script: ./tools/azure-pipelines/setup_kubernetes.sh
     - script: M2_HOME=/home/vsts/maven_cache/apache-maven-3.2.5/ PATH=/home/vsts/maven_cache/apache-maven-3.2.5/bin:$PATH ${{parameters.environment}} STAGE=compile ./tools/azure_controller.sh compile
       displayName: Build
+    - script: FLINK_DIR=build-target ./flink-end-to-end-tests/run-pre-commit-tests.sh
```

Comment:
add a TODO to remove this since we're pretty much eliminating the concept of pre-commit e2e tests.

Reply:
Very good point, thanks!